### ISSUE TITLE
The image built through the Dockerfile cannot start properly.

### ISSUE THREAD

**Issue by xiemeilong** [autonomousvision/sdfstudio#129]
**Describe the bug**
libQt5Core.so.5: version `Qt_5.15' not found (required                
           by                                                                                                           
           /home/user/.local/lib/python3.8/site-packages/pymeshlab/pmeshlab.cpython-38-x86_64-linux-gnu.s               
           o)                                                                                    

```
✔ Wrote new completion to /home/user/nerfstudio/scripts/completions/bash/_ns-process-data!     install.py:119
[14:30:14] ❌ Completion script generation failed: ['ns-extract-mesh', '--tyro-print-completion', 'bash'] install.py:109
           Traceback (most recent call last):                                                             install.py:113
             File "/home/user/.local/bin/ns-extract-mesh", line 5, in <module>                                          
               from scripts.extract_mesh import entrypoint                                                              
             File "/home/user/nerfstudio/scripts/extract_mesh.py", line 17, in <module>                                 
               from nerfstudio.utils.marching_cubes import (                                                            
             File "/home/user/nerfstudio/nerfstudio/utils/marching_cubes.py", line 4, in <module>                       
               import pymeshlab                                                                                         
             File "/home/user/.local/lib/python3.8/site-packages/pymeshlab/__init__.py", line 11, in                    
           <module>                                                                                                     
               from .pmeshlab import *                                                                                  
           ImportError: /usr/lib/x86_64-linux-gnu/libQt5Core.so.5: version `Qt_5.15' not found (required                
           by                                                                                                           
           /home/user/.local/lib/python3.8/site-packages/pymeshlab/pmeshlab.cpython-38-x86_64-linux-gnu.s               
           o)                                                                                                           
                                                                                                                        
           ✔ Wrote new completion to /home/user/nerfstudio/scripts/completions/bash/_ns-render!           install.py:119
           ✔ Wrote new completion to /home/user/nerfstudio/scripts/completions/bash/_ns-eval!             install.py:119
           ✔ Wrote new completion to /home/user/nerfstudio/scripts/completions/bash/_ns-download-data!    install.py:119
           ✔ Wrote new completion to /home/user/nerfstudio/scripts/completions/bash/_ns-render-mesh!      install.py:119
[14:30:15] ❌ Completion script generation failed: ['ns-train', '--tyro-print-completion', 'bash']        install.py:109
           Traceback (most recent call last):                                                             install.py:113
             File "/home/user/.local/bin/ns-train", line 5, in <module>                                                 
               from scripts.train import entrypoint                                                                     
             File "/home/user/nerfstudio/scripts/train.py", line 50, in <module>                                        
               from nerfstudio.configs.method_configs import AnnotatedBaseConfigUnion                                   
             File "/home/user/nerfstudio/nerfstudio/configs/method_configs.py", line 57, in <module>                    
               from nerfstudio.models.dto import DtoOModelConfig                                                        
             File "/home/user/nerfstudio/nerfstudio/models/dto.py", line 54, in <module>                                
               from nerfstudio.utils.marching_cubes import get_surface_occupancy                                        
             File "/home/user/nerfstudio/nerfstudio/utils/marching_cubes.py", line 4, in <module>                       
               import pymeshlab                                                                                         
             File "/home/user/.local/lib/python3.8/site-packages/pymeshlab/__init__.py", line 11, in                    
           <module>                                                                                                     
               from .pmeshlab import *                                                                                  
           ImportError: /usr/lib/x86_64-linux-gnu/libQt5Core.so.5: version `Qt_5.15' not found (required                
           by                                                                                                           
           /home/user/.local/lib/python3.8/site-packages/pymeshlab/pmeshlab.cpython-38-x86_64-linux-gnu.s               
           o)                                                                                                           
                                                                                                        
```

**Screenshots**

**Comment by xiemeilong** [comment]
@niujinshuchong Is there any difference between installing in Dockerfile and installing after running? Also, I need to use bakesdf to generate models.

**Comment by niujinshuchong** [comment]
Hi, installing in Dockerfile and installing after running should be the same but you have more control in the latter one since you have more control and easily check what's wrong. You could also move the import of pymeshlab here https://github.com/autonomousvision/sdfstudio/blob/master/nerfstudio/utils/marching_cubes.py#L162 so it's imported only when it's needed.

**Comment by xiemeilong** [comment]
@niujinshuchong After I moved the "import pymeshlab" statement and included the "cryptography" dependency in the pyproject.toml file, the image was able to start successfully. Thank you very much. 
By the way, I have a question to ask you. I have just started to learn about NERF and SDF, and I'm curious why the SDFStudio dataset requires the use of Omnidata to generate depth and normals. I didn't notice such requirement in the model paper.
